src/relion/zocalo/extract.py

```diff
@@ -264,22 +264,68 @@ def dummy(self, *args, **kwargs):
                 )
                 extract_width = round(extract_params.relion_options.small_boxsize / 2)
 
-            # Background normalisation
-            if extract_params.norm:
-                # Distance of each pixel from the centre, compared to background radius
-                grid_indexes = np.meshgrid(
-                    np.arange(2 * extract_width),
-                    np.arange(2 * extract_width),
+            # Distance of each pixel from the centre, compared to background radius
```

Comment:
Do we want an option to turn this off, or is it essential enough to always be on?

Reply:
I think we'll leave it permanently on for now